python/python/Ice/Connection.py

```diff
@@ -2,23 +2,11 @@
 #
 # Copyright (c) ZeroC, Inc. All rights reserved.
 #
```

Comment:
I think we're using single line copyright headers now.
```suggestion
# Copyright (c) ZeroC, Inc. All rights reserved.
```

Reply:
We should deal with simplifying copyrights in a separate PR. We are still using multiline copyrights in main.